### DISCUSSION TITLE
myBMW subscription

### DISCUSSION THREAD

**Discussion by zAFierCe** [BimmerGestalt/AAIdrive#586] [Q&A] [answered]
So my subscription to mybmw expired a few days ago and suddenly aaidrive isnt connecting anymore do i need an active bmw subscription?

**Comment by bogdan-calapod** [comment, marked as answer]
Yes, the Apps feature that AAIDrive is based on requires an active subscription.

**Comment by TBGR** [comment]
Or you can have someone code the BMW Apps option if you have iDrive4/5. Still needs an FSC code though.
